tasks/main.yml

```diff
@@ -53,6 +68,24 @@
     - kubeadm
     - kubectl
 
+- name: Check that candidate minor version equals {{ kubernetes_version }}
+  shell: |-
+    line="$(apt-cache policy {{ item | quote }} | grep '  Installed: ')"
```

Comment:
Shouldn't this grep for `Candidate` instead of `Installed`?

Reply:
right